Docs/User Guide.md

```diff
@@ -0,0 +1,88 @@
+## Overview
+
+Many HIL test systems require hardware fault injection to simulate faults between the ECU and the rest of the system. The **Routing and Faulting Custom Device** injects faults using routing modules. The **SLSC Switch Custom Device** provides hardware support for SLSC Switch routing modules.
```

Comment:
Should this be "DUT" throughout, instead of "ECU"?

Reply:
I am thinking of the LRU use case.

Reply:
Agreed. We should replace ECU with DUT....wherever it make sense.

Reply:
I replaced all instances of ECU with DUT.